<?xml version="1.0"?>

<launch>

    <node pkg="heron_mission" type="planner" name="planner">
        <rosparam file="$(find heron_mission)/config/heron_mission.yaml" />
    </node>
    <node pkg="heron_mission" type="model_manager.py" name="model_manager" />

    <!-- Launch RVIZ with a configuration file -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find heron_mission)/rviz/debug.rviz" />

</launch>